src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/ApplicationIntegrationTest.kt

```diff
@@ -82,13 +96,20 @@ class ApplicationIntegrationTest(@Autowired val jwtAuthHelper: JwtAuthHelper) :
       .get()
```

Comment:
I'm not sure about this approach to test multiple things in a single test, I'm missing the test name as documentation - might be better to create a @Nested inner class for each suite of tests, with a single test for each scenario?